tests/unit_tests/crypto_tests.cpp

```diff
@@ -356,7 +356,7 @@ int fe_cmp(const fe a, const fe b)
 static const fe scalar_L_fe = { 16110573, 10012311, -6632702, 16062397, 5471207, 0, 0, 0, 0, 4194304 };
 
 
-__declspec(align(32))
```

Comment:
I'm not sure that removing this line is a good idea, even tho it's just a tests, but i guess @sowle  can give us more input on that

Reply:
Not an issue after rebase